guides/hugo-modules/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+_schema: default
+_uuid: b42c9e99-49f4-4129-bfc4-0e6eaff956ed
+_created_at: 2023-06-07T00:00:00Z
+title: Introduction
+nav_title: Introduction
+order: 1
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
+description: Learn how to connect Hugo Modules to your CloudCannon site.
+tags: []
+related_articles: []
+related_links: []
+---
+Welcome to the [Hugo Modules](https://gohugo.io/hugo-modules/) guide. If you want to use themes and components from other repositories on your site, you are in the right place! To configure CloudCannon so that your Hugo modules install correctly, you must customize how CloudCannon loads your modules into your build.
+
+<comp.Notice info_type="important">
+  This guide assumes you already have a Hugo site on CloudCannon and are using GitHub as your module repository.
+</comp.Notice>
+
+Hugo Modules can use public or private repositories.
+
+For a public repository, you won't require any configuration in CloudCannon to build successfully. However, you might want to improve your experience by using CloudCannon's [Site Mounting](/documentation/articles/site-mounting/) feature. With Site Mounting, a new build of your site is triggered every time the Hugo Module is updated.
```

Comment:
Should this link be to the site mounting step of this guide, rather than the site mounting feature itself? Would be a more contextually useful place to land if they click it.

Reply:
(and for the paragraph right below this)